### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by mel-d**
I'm getting an undefined method error on current_user.

### CODE REVIEW

**Comment by mel-d** [comment]
I know I need to create another column for the theater, but I'm assuming this should be the theater id#, right? As well as with the award and play columns, I know good practice is to have the most foreign ids possible in a db table... but is there a way the user can type a string into an actual text field and then have it converted into id# in the db? I feel like that would add lots of opportunity for errors if things aren't typed perfectly... should I make all three of those columns the names instead of ids?

**Comment by mel-d** [comment]
^That's barely coherent. Sorry.

**Comment by mel-d** [comment]
Crap, and I forgot to re-add the user id as a param in the model. Doing that rn.

**Comment by mel-d** [comment]
The user id worked that time! Question about foreign keys vs. names still stands

File: app/models/user.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ class User < ActiveRecord::Base
   validates :first_name, presence: true
   validates :last_name, presence: true
 
-  def record_nominations(nominations_hash)
+  def record_nominations(user_id, nominations_hash)
```
Comment (sumeetjain) [inline app/models/user.rb:13]: Has approx 8 statements (TooManyStatements)

Comment (sumeetjain) [inline app/models/user.rb:13]: Method has too many lines. [15/10]

Comment (sumeetjain) [inline app/models/user.rb:13]: Inconsistent indentation detected.

**Comment by sumeetjain** [comment]
Theater, Show, Nominee, and Role should _all_ be string columns. They're not related to foreign keys at all. We can't rely on TAG pre-populating things ahead of time. The simple solution is free-text entry.

File: app/models/user.rb (hunk 1 above)
Comment (sumeetjain) [inline app/models/user.rb:13]: This is redundant. You're in the User model, so just call `self`. That represents whatever User object is having the method run on it.

**Pull request merged**